server/aws-lsp-codewhisperer/src/language-server/inline-completion/session/sessionManager.ts

```diff
@@ -73,6 +75,7 @@ export class CodeWhispererSession {
     customizationArn?: string
     includeImportsWithSuggestions?: boolean
     codewhispererSuggestionImportCount: number = 0
+    flareRequestId?: string // Add flareRequestId field
```

Comment:
Lets remove flareRequestId. This construct was created for debugging in the dev branch.

Reply:
ack